ltm.go

```diff
@@ -1738,11 +1753,14 @@ func (b *BigIP) PoolMemberStatus(pool string, member string, state string, owner
 }
 
 // CreatePool adds a new pool to the BIG-IP system by name.
-func (b *BigIP) CreatePool(name string) error {
+func (b *BigIP) CreatePool(name string, partition string) error {
 	config := &Pool{
 		Name: name,
 	}
 
+	if len(partition) > 0 {
+		config.Partition = partition
```

Comment:
same here, no possibility to without partition specified in POST.

Reply:
So if you need to Create a Pool to specify a partition -- look at  AddPool

```go
f5Pool = &bigip.Pool{
    Name:      f5PoolName,
    Partition: s.partition,
}
err = s.f5.AddPool(f5Pool)
```